demo/index.html

```diff
@@ -4,7 +4,6 @@
 		<meta charset="utf-8">
```

Comment:
I left this file the messy Polymer demo style so we can test the old Polymer mixins with messy Polymer style things